### ISSUE TITLE
Create PickStackActivity

### ISSUE THREAD

**Issue by stefan-niedermann** [stefan-niedermann/nextcloud-deck#358]
## Background

To create new cards we need some information like `accountId`, `boardId` and `stackId`.

In the default workflow one would open the app, navigate to the place where a new card should be created and then hits the "create"-button. From this position all the needed information can be passed to the `EditActivity`.

## Problem with current implementation

When using android shortcut or the tile, we don't have the information and need to pick them before we can succesfully create a new card. We are currently doing this within the `EditActivity` (*this behavior is currently completely broken*).

This issue is about moving the gathering part into an own `PickStackActivity` because of separation of concerns and to simplify the `EditActivity` which is currently very complicated due to the asynchronous dependeny of the dataflow.

## Goal

Goal is to open the `PickStackActivity` which provides the necessairy controls (maybe prepopulated select-boxes for account, board and stack?) and starts on submit the `EditActivity` and passes the gathered information directly to it.

## Work in progress

- [x] Create new Activity
- [x] Provide interface to select required parameters
- [x] Save selected values
- [x] Preselect saved values for `account`, `board` and `stack`
- [x] Start `EditActivity`
- [x] Hide `Boards` and `Stacks` without `edit`-permission
- [x] Handle no available boards / stacks
- [x] Fix android shortcut
- [x] Clean up `EditActivity`
